mission/simulator/opssat-spacecraft-simulator/src/main/java/opssat/simulator/threading/SimulatorNode.java

```diff
@@ -3799,7 +3800,7 @@ public CommandResult runGenericMethodForCommand(CommandDescriptor c)
           String hour = hms[0];
           String minute = hms[1];
           String ms = hms[2];
-          ms = ms.replace("\\.", "");
+          ms = ms.replaceAll("\\.", "");
```

Comment:
@mercom I'll approve the changes since the test you added seem straightforward but I'm just puzzled by this change, why was there multiple dot in the utc ms field as a float format?

Reply:
@TanguySoto  It's not that it had multiple dots, but that the replace("\\.", "") doesn't behave the way it was expected in this code.
String replace and replaceAll behave slightly differently, as both replace all occurrences, the first one uses simple charsequence while the latter uses regex expression. Hence when using the replace() method we should not escape the dot, but in replaceAll it is required.

Here we could either use ms.replaceAll("\\.", "") or ms.replace(".","").

Reply:
I can change it to  ms.replace(".","") if you believe it would be easier to read and understand?

Reply:
@mercom Thanks for the clear explanation! I think this is fine. I'll merge!